scripts/BadgeManager.js

```diff
@@ -2,20 +2,38 @@
 
 'use strict';
 
-function BadgeManager(addEventListener) {
+function BadgeManager(addEventListener, isEnabled) {
 
 	addEventListener("hourMusic", function(hour) {
-		chrome.browserAction.setBadgeText({ text: formatHour(hour) });
+		if(isEnabled()) {
+			chrome.browserAction.setBadgeText({ text: formatHour(hour) });
+		}
+		playIcon();
 	});
 
 	addEventListener("kkStart", function() {
-		chrome.browserAction.setBadgeText({ text: "KK" });
+		if(isEnabled()) {
+			chrome.browserAction.setBadgeText({ text: "KK" });
+		}
+		playIcon();
 	});
 
 	addEventListener("pause", function() {
 		chrome.browserAction.setBadgeText({ text: "" });
+		pauseIcon();
 	});
+	
+	function playIcon() {
+		chrome.browserAction.setIcon({
+			path : "img/icon_38_leaf-02.png"
```

Comment:
Is the play icon supposed to show the pause symbol and the pause icon supposed to show the play symbol? I can see that it might be saying that while music is playing, it functions as a pause button, but I feel like the other way around would be more intuitive. That's just a matter of taste though.